tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -193,6 +193,19 @@ Verify JupyterHub Pod Logs Dont Have Errors About Distutil Library
     ...       ODS-586
     Verify Errors In Jupyterhub Logs
 
+Verify Grafana Is Connected To Prometheus Using TLS
+    [Documentation]    Verifies Grafana is connected to Prometheus using TLS
+    [Tags]    Tier2
+    ...       ODS-963
+    [Setup]  Set Library Search Order  Selenium Library
+    ${res} =    Run
+    ...    oc get secret/grafana-datasources -n redhat-ods-monitoring -o json | jq -r '.data["datasources.yaml"]' | base64 -d | jq '.datasources[0] | .jsonData.tlsSkipVerify'
```

Comment:
would it be possible to use openshift-library here? I think you could extract `data["datasources.yaml"]` using Oc Get and the filters. Then, using a couple of line of code you should be able to extract the remaining data

Reply:
Using OpenShiftLibrary , I tried to decode the secret after extraction using `Evaluate    base64.b64decode(${res[0]['data']['datasources.yaml']})  modules=base64` It says `failed: SyntaxError: unexpected EOF while parsing`

Reply:
Updated.